pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -139,6 +139,7 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 		cidr.IP.To4()[3] += 1
 
 		json := fmt.Sprintf(NET_CONFIG_TEMPLATE, BridgeName, plugin.MTU, network.DefaultInterfaceName, podCIDR, cidr.IP.String())
+		glog.V(2).Infof("CNI network config set to %v", json)
```

Comment:
Check this: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/kubenet/kubenet_linux.go#L122
It only process podCIDR change event. This is not a good code structure, tough. But we can fix it later.